src/z3c/recipe/sphinxdoc/__init__.py

```diff
@@ -168,9 +171,11 @@ def install(self):
     update = install
 
 
-def main(projects):
+def main(projects, argv=None, exit_on_error=False):
```

Comment:
I was wrong: `sphinx.main()` doesn't call `sys.exit()`, it just returns the result of `sphinx.build_main()`.

Never mind!